.gitignore

```diff
@@ -24,3 +24,5 @@ tests/test_output/*
 .DS_Store
 .hypothesis
 
+tests/test_data/dstability/Tutorial_serialized_new.stix
+tests/DStabilityConsole
```

Comment:
Why have these been added? When will these be generated in the _sources_ folder? This will be forgotten once it is no longer necessary, resulting in clutter in the repo.

Reply:
The way the geolib code is currently set up is that some tests create data during runtime and the console is included inside the repo as well. This first line can be improved with a better test output path in a temp folder as part of a separate issue.

The second line will remain relevant for a while as long as the config of the tests remains to be to search in the tests folder for the console

Reply:
Alright, that explains it. Since the reason for the second line is also covered by documentation, that is acceptable.